tests/console/copy_iso_to_external_drive.pm

```diff
@@ -43,7 +43,7 @@ sub run {
 sub post_run_hook {
     #prepare environment for next test
     type_string "logout\n";
-    assert_screen "text-login";
+    assert_screen "tty6-selected";
```

Comment:
please don't hard-code the tty$nr-selected in the test modules. This is what we have the `select_console`-calls for. So, let me think, why do we have this `assert_screen` here?

commit 6384c26c
Author: Joachim Rauch <jrauch@suse.com>
Date:   Fri Aug 18 12:41:57 2017 +0200

    Fix ISO_IN_EXTERNAL test
    
    Fixed the problem when rebooting to the installation.

introduced it. That was in PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3425 . Unfortunately the PR only states "Fixed the problem when rebooting to the installation" without stating in detail what actually was the problem. Honestly I think this line can simply be removed. It should not matter if the actual "logout" appears in the background while we already switch the x11 console